### DISCUSSION TITLE
Vercel Ecommerce Not Showing Updated Shopify Products

### DISCUSSION THREAD

**Discussion by petermazzocco** [vercel/vercel#10983] [Help]
### Question

Hello!

I recently made an ecommerce site using the Vercel ecommerce repo and Shopify as the CMS. But I am adding new products to Shopify that are not populating on the site. I was wondering if someone could help me out with figuring out why this is happening?

For example:

When I go directly to the new products page -> https://www.thedigitaldam.com/product/the-analog-dam-off-white it will populate and show the product.

But when I go to the collection it is apart of -> https://www.thedigitaldam.com/collections/the-analog-dam It says no products found in this collection

But on my localhost, it shows the products within the collection just fine. 

Is this an issue of Shopify taking awhile to populate the products on the domain vs local?

**Comment by Superallyman** [comment]
Did you find an answer to why this was happening to your deployed app?

**Reply by petermazzocco** [reply]
Yes! It was the auto caching set to one day. I removed it and set it to zero

**Comment by Superallyman** [comment]
Thanks for the response! Your suggestion was helpful
